hugegraph-test/src/main/java/com/baidu/hugegraph/core/VertexCoreTest.java

```diff
@@ -6667,6 +6669,36 @@ public void testQueryByPageWithInvalidPage() {
         });
     }
 
+    @Test
+    public void testQueryByPageWithSpecialBase64Chars() {
+        final String pageWith3Base64Chars = "AAAAADsyABwAEAqI546LS6WW57unBgA" +
```

Comment:
delete these lines, as already moved to unit test

Reply:
keep them for tests below